src/test/scala/mesosphere/marathon/MarathonConfTest.scala

```diff
@@ -184,33 +185,15 @@ class MarathonConfTest extends UnitTest {
       assert(conf.acceptedResourceRolesDefaultBehavior() == AcceptedResourceRolesDefaultBehavior.Unreserved)
     }
 
-    "--accepted_resource_roles_default_behavior overrides --default_accepted_resource_roles *" in {
-      val conf = MarathonTestHelper.makeConfig(
+    "throw an exception when both --accepted_resource_roles_default_behavior and --default_accepted_resource_roles are specified" in {
```

Comment:
If we merge this, we need to adjust the marathon.sh launch script in DC/OS. Maybe. 

At the moment, it allows both to be specified:
If `default_accepted_resource_roles` is set, but `accepted_resource_roles_default_behavior` is not, it sets a default for `accepted_resource_roles_default_behavior`, but doesn't clear out the `default_accepted_resource_roles`

Reply:
As discussed in Slack, the launch script in DC/OS only sets the latter (`--accepted_resource_roles_default_behavior`) if the former is not set (`--default_accepted_resource_roles`)